include/http/request_handler.js

```diff
@@ -606,6 +650,13 @@ module.exports = function RequestHandlerModule(pb) {
         //set the session
         this.session = session;
 
+        //set the site -- how do we handle improper sites here?
+        //TODO Handle global differently here when we pull through global site designation
+        this.siteObj = RequestHandler.sites[this.hostname]
+            ? RequestHandler.sites[this.hostname]
+            : this.serve404();
+        this.site = this.siteObj.uid;
+        this.siteName = this.siteObj.displayName;
```

Comment:
Yes removing it now.

Ben Tidwell
Software Engineer III | Talent Network
CareerBuilder
5550-A Peachtree Pkwy, Norcross GA 30092

From: Brian <notifications@github.com<mailto:notifications@github.com>>
Reply-To: pencilblue/pencilblue <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Tuesday, July 14, 2015 at 5:19 PM
To: pencilblue/pencilblue <pencilblue@noreply.github.com<mailto:pencilblue@noreply.github.com>>
Cc: Ben Tidwell <ben.tidwell@careerbuilder.com<mailto:ben.tidwell@careerbuilder.com>>
Subject: Re: [pencilblue] Multi Tenancy (#680)


In include/http/request_handler.js<https://github.com/pencilblue/pencilblue/pull/680#discussion_r34620990>:

> @@ -606,6 +650,13 @@ module.exports = function RequestHandlerModule(pb) {
>          //set the session
>          this.session = session;
>
> +        //set the site -- how do we handle improper sites here?
> +        //TODO Handle global differently here when we pull through global site designation
> +        this.siteObj = RequestHandler.sites[this.hostname]
> +            ? RequestHandler.sites[this.hostname]
> +            : this.serve404();
> +        this.site = this.siteObj.uid;
> +        this.siteName = this.siteObj.displayName;


Has the TODO already been handled?

-
Reply to this email directly or view it on GitHub<https://github.com/pencilblue/pencilblue/pull/680/files#r34620990>.